src/aspire/utils/filters.py

```diff
@@ -197,7 +216,7 @@ def __init__(self, xfer_fn_array):
     def _evaluate(self, omega):
         sz = self.sz
         # TODO: This part could do with some documentation - not intuitive!
-        omega = omega / self._scale
+        omega = omega
```

Comment:
No point of this anymore

Reply:
Good catch. removed in the rebased commit a95d02f.